### DISCUSSION TITLE
New refrigerants on V9.1

### DISCUSSION THREAD

**Discussion by LidiaLLP** [usnistgov/REFPROP-issues#639] [General]
Do you have a patch for new predifined mixtures like R515B,  R471A and all the new A2L refrigerants. I am interested for a patch on REFPROP 9.1

**Comment by ianhbell** [comment]
You will need to upgrade to 10 to get the new fluid file for R-1234yf, but the mixture model format has not changed.  What "all the new A2L refrigerants" are in your list?
